utils/ci/corefile_in_gha.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+cd daos
+
+echo ::group::Check for corefiles
+if [ $BASE_DISTRO == "ubuntu" ]
+then
+    # "apport" corefile repo
+    COREFILE_DIR="/var/crash/"
+else
+    COREFILE_DIR="/var/lib/systemd/coredump/"
+fi
+if [ $(ls -1 /var/lib/systemd/coredump/ | wc -l) == 0 ]
```

Comment:
(lint) Quote this to prevent word splitting. [SC2046]
(lint) Use find instead of ls to better handle non-alphanumeric filenames. [SC2012]